CHANGELOG.md

```diff
@@ -1,4 +1,93 @@
-### 1.0.0 @ May 14 2015
+## 2.0.0
+
+### Summary
+
+There has been quite some work done for this new major version:
+
+1. Make `levelup` more generic by reducing focus on [`leveldown`](https://github.com/Level/leveldown) and [`LevelDB`](https://github.com/google/leveldb).
+2. Make `levelup` more generic by removing code related to encodings, which would allow *down implementations to manage encodings themselves.
+3. Use [`standard`](https://github.com/standard/standard) as linter to hopefully reduce friction when accepting new PRs. It's not really about which linter to use, but rather more about having a linter at all and many of us also use `standard` in other projects.
```

Comment:
- Remove "hopefully", don't we already know it to be true? :)
- Second sentence sounds somewhat defensive. Could just leave it out, standard's readme already has [a good motivation](https://github.com/standard/standard#i-disagree-with-rule-x-can-you-change-it). Or summarize that as "to avoid bikeshedding".

Reply:
Agreed. I knew I would get help with the language later, just wanted to write _something_ up first :) I'll fix.